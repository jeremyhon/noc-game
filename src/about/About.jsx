import React from 'react'
import './About.css'

export default class About extends React.Component {
  render() {
    return (
      <div className="about">
        Have you ever thought that your internship program was poorly managed? <br />
        And you thought to yourself "I can do better" <br />
        <br/>
        Well.<br/>
        <br/>
        Here you go.<br/>
        <br />
        Created by <a href="http://jeremyhon.github.io">Jeremy Hon</a><br />
        <br />
        <a href="https://www.github.com/jeremyhon/noc-game">GitHub Repo</a>
      </div>
    )
  }
}
