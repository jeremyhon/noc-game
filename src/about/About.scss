.about {
  text-align: left;
  margin: auto;
  padding: 10px;
  max-width: 600px;
}
