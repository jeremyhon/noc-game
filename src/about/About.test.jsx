import React from 'react'
import { shallow } from 'enzyme'

import About from './About'

it('matches the snapshot', () => {
  expect(shallow(<About />)).toMatchSnapshot()
})
