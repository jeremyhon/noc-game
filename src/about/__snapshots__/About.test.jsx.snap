// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="about"
>
  Have you ever thought that your internship program was poorly managed? 
  <br />
  And you thought to yourself "I can do better" 
  <br />
  <br />
  Well.
  <br />
  <br />
  Here you go.
  <br />
  <br />
  Created by 
  <a
    href="http://jeremyhon.github.io"
  >
    Jeremy Hon
  </a>
</div>
`;
